src/mantine-dates/src/components/TimeRangeInput/TimeRangeInput.styles.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  createMemoStyles,
+  MantineTheme,
+  INPUT_SIZES,
+  MantineSize,
+  getSizeValue,
+} from '@mantine/core';
+
+interface TimeRangeInputStyles {
+  theme: MantineTheme;
+  size: MantineSize;
+}
+
+const inputSizes = {
+  xs: 20,
+  sm: 23,
+  md: 25,
+  lg: 27,
+  xl: 29,
+};
+
+export default createMemoStyles({
+  disabled: {
+    opacity: 0.6,
+    cursor: 'not-allowed',
+  },
+
+  wrapper: {
+    paddingTop: '5px',
```

Comment:
1. Why wrapper has padding top?
2. When you add px values, set value as number: `paddingTop: 5`, not `paddingTop: '5px'`